app/views/users/show.html.erb

```diff
@@ -1 +1,38 @@
-<h2>Details of a given user</h2>
\ No newline at end of file
+<%= stylesheet_link_tag 'users_show', media: 'all', 'data-turbolinks-track': 'reload' %>
+
+<div class="user-index">
+  <div class="user-container">
+    <img src="<%= @user.photo %>" alt="" class="photo">
+
+    <div class="name-posts">
+      <div class="name">
+        <%= @user.name %>
+      </div>
+      <div class="posts">
+        <p>Number of posts: <%= @user.posts.count %></p>
+      </div>
+    </div>
+  </div>
+
+  <div class="user-bio-container">
+    <p class="bio-title">Bio</p>
+    <p class="bio-content"><%= @user.bio %></p>  
+  </div>
+
+<div class="user-posts">
+  <% @two_more_recent_posts.each do |post| %>
```

Comment:
Great job in creating user-friendly views that matches the wireframe. 💯 

I noticed that there is a missed requirement in the project. The requirements states that the user show page should use the `recent_post` method to render the user's three most recent posts. Instead, a new variable `@two_more_recent_posts` was created in the `users_controller.rb` show action, which only renders two recent posts instead of three.

### Sugestion

Since the `three_most_recent_posts` public method already exist in the user model. There's no need to create a new variable to perform this action

Kindly consider this aproach:

```ruby
<% @user.three_most_recent_posts.each do |post| %>
```

Overall, great acknowledgement to the hard work and dedication in creating functional views for the project. I am confident that with this slight adjustment, you will be able to meet all the requirements successfully.

Keep up the great work! 🚀